.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  extends: ["plugin:vue/essential", "@vue/prettier"],
```

Comment:
Lets use [standardjs](https://standardjs.com/) rules later.
I'll create a new issue for it.